dev/PushNotifications/PushNotificationReceivedEventArgs.cpp

```diff
@@ -132,16 +132,36 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         }
     }
 
-    std::string PushNotificationReceivedEventArgs::Utf16ToUtf8(_In_z_ PCWSTR utf16)
+    std::string PushNotificationReceivedEventArgs::Utf16ToUtf8(_In_ std::wstring const& utf16string)
```

Comment:
Nit: Should be a global function IMHO and not part of PushNotificationREceivedEventArgs

Reply:
Agree. Let's have a wpnlib for WAS 😊.